ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -187,7 +187,9 @@ objects:
 
         # set up openstack env vars for testing
         function get_clouds_param() {
-          awk '/^\s+'$1':/ { print $2 }' /tmp/cluster/clouds.yaml | sed -e 's/^"//' -e 's/"$//'
+          key=$1
+          default=$2
+          awk '/^\s+'$key':/ { print $2 }' /tmp/cluster/clouds.yaml | sed -e 's/^"//' -e 's/"$//' | grep . || echo $default
```

Comment:
Won't this mask the error if required fields are not found?

Reply:
It will return an empty string, same as before. The difference is that now you can set a default for when the key is not found.

Reply:
Perfect; I had missed the addition of `|grep .`.